packages/cli/cloud/src/services/cli-api.ts

```diff
@@ -2,35 +2,42 @@ import axios, { type AxiosResponse } from 'axios';
 import fse from 'fs-extra';
 import os from 'os';
 import { apiConfig } from '../config/api';
-import type { CLIContext, CloudCliConfig } from '../types';
+import type { CLIContext, CloudCliConfig, TrackPayload } from '../types';
 import { getLocalConfig } from '../config/local';
 
 import packageJson from '../../package.json';
 
 export const VERSION = 'v1';
 
 export type ProjectInfos = {
+  id: string;
   name: string;
-  nodeVersion: string;
-  region: string;
+  displayName?: string;
+  nodeVersion?: string;
+  region?: string;
   plan?: string;
   url?: string;
 };
+
 export type ProjectInput = Omit<ProjectInfos, 'id'>;
 
 export type DeployResponse = {
   build_id: string;
   image: string;
 };
 
-export type TrackPayload = Record<string, unknown>;
-
 export type ListProjectsResponse = {
   data: {
     data: string;
   };
 };
 
+export type ListLinkProjectsResponse = {
+  data: {
+    data: ProjectInfos[];
```

Comment:
data can either be `ProjectInfos[]` or an empty object `{}` in case there is no project.

Currently we get an error if we have no project:

```
✖ An error occurred while fetching your projects from Strapi Cloud.
[DEBUG] Failed to list projects TypeError: projectList.filter is not a function
    at getProjectsList (/Users/abdallah/Documents/git/strapi/packages/cli/cloud/dist/index.js:1282:34)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async action$2 (/Users/abdallah/Documents/git/strapi/packages/cli/cloud/dist/index.js:1335:20)

```

Reply:
Weird, tested that 🤔 maybe introduced in last commits